hedera-node/src/main/java/com/hedera/services/store/contracts/precompile/codec/EncodingFacade.java

```diff
@@ -56,38 +66,41 @@
 public class EncodingFacade {
     public static final Bytes SUCCESS_RESULT = resultFrom(SUCCESS);
     private static final long[] NO_MINTED_SERIAL_NUMBERS = new long[0];
-//    private static final String STRING_RETURN_TYPE = "(string)";
-//    public static final String UINT256_RETURN_TYPE = "(uint256)";
-//    public static final String BOOL_RETURN_TYPE = "(bool)";
-//    private static final TupleType mintReturnType = TupleType.parse("(int32,uint64,int64[])");
-//    private static final TupleType burnReturnType = TupleType.parse("(int32,uint64)");
-//    private static final TupleType createReturnType = TupleType.parse("(int32,address)");
-//    private static final TupleType totalSupplyType = TupleType.parse(UINT256_RETURN_TYPE);
-//    private static final TupleType balanceOfType = TupleType.parse(UINT256_RETURN_TYPE);
-//    private static final TupleType allowanceOfType = TupleType.parse(UINT256_RETURN_TYPE);
-//    private static final TupleType hapiAllowanceOfType = TupleType.parse("(int32,uint256)");
-//    private static final TupleType approveOfType = TupleType.parse(BOOL_RETURN_TYPE);
-//    private static final TupleType hapiApproveOfType = TupleType.parse(INT_BOOL_PAIR_RETURN_TYPE);
-//    private static final TupleType hapiApproveNftType = TupleType.parse("(int32)");
-//    private static final TupleType decimalsType = TupleType.parse("(uint8)");
-//    private static final TupleType ownerOfType = TupleType.parse("(address)");
-//    private static final TupleType getApprovedType = TupleType.parse("(address)");
-//    private static final TupleType hapiGetApprovedType = TupleType.parse("(int32,address)");
-//    private static final TupleType nameType = TupleType.parse(STRING_RETURN_TYPE);
-//    private static final TupleType symbolType = TupleType.parse(STRING_RETURN_TYPE);
-//    private static final TupleType tokenUriType = TupleType.parse(STRING_RETURN_TYPE);
-//    private static final TupleType ercTransferType = TupleType.parse(BOOL_RETURN_TYPE);
-//    private static final TupleType isApprovedForAllType = TupleType.parse(BOOL_RETURN_TYPE);
-//    private static final TupleType isTokenKycType = TupleType.parse(INT_BOOL_PAIR_RETURN_TYPE);
-//    private static final TupleType hapiIsApprovedForAllType =
-//            TupleType.parse(INT_BOOL_PAIR_RETURN_TYPE);
-//    private static final TupleType getTokenDefaultFreezeStatusType =
-//            TupleType.parse(INT_BOOL_PAIR_RETURN_TYPE);
-//    private static final TupleType getTokenDefaultKycStatusType =
-//            TupleType.parse(INT_BOOL_PAIR_RETURN_TYPE);
-//    private static final TupleType isTokenFrozenType = TupleType.parse(INT_BOOL_PAIR_RETURN_TYPE);
-//    private static final TupleType isTokenType = TupleType.parse(INT_BOOL_PAIR_RETURN_TYPE);
-//    private static final TupleType getTokenType = TupleType.parse("(int32,int32)");
+    //    private static final String STRING_RETURN_TYPE = "(string)";
```

Comment:
Is there a need for this commented code?  Should it be removed?

Reply:
Please remove if no longer needed